de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/concepts/util/ActiveConceptHelper.java

```diff
@@ -188,6 +188,19 @@ private static Concept getConceptForEObject(EObject conceptType) {
 		return (currentType instanceof Concept) ? (Concept) currentType : null;
 	}
 	
+	/**
+	 * Method to get the concept name to display in concept selection dialog
+	 * @param concept the concept which name details are needed
+	 * @return the display name plus FQN and version or the FQN and version when there is no display name
+	 */
+	public static String getConceptName(Concept concept) {

```

Comment:
The method name is a bit misleading since concept has a "name" property, but we do more than just getting that here.
I would propose something like "getConceptNameWithVersion"